src/stacks-hierarchy/Root_ShareTravelHabitsScreen.tsx

```diff
@@ -24,8 +26,28 @@ export const Root_ShareTravelHabitsScreen = () => {
 
   const {onboardForBeacons} = useBeaconsState();
 
+  const analytics = useAnalytics();
+
+  // call useFocusEffect to send analytics once when the screen is shown
+  useFocusEffect(
+    useCallback(() => {
+      analytics.logEvent(
+        'Onboarding',
+        'didSeeShareTravelHabitsScreen',
+      );
+    }, [analytics]),
+  );
```

Comment:
No biggie, but I think it should instead be a `useEffect` here.

I would mainly use `useFocusEffect` if we explicitly wanted something to be run only **while the component is focused**. If we want something to run only **when the component becomes focused**, i would use a focus event listener. If we want something to be run **only once when mounting**, then I would use `useEffect` with empty dependency array (or in this case, with just `analytics`, since it is stable).

If the answer to "Do we want to send the analytics event every time the component becomes focused" is no, then we shouldn't be using `useFocusEffect`.